packages/hardhat-core/src/internal/core/config/config-validation.ts

```diff
@@ -430,7 +431,7 @@ export function getValidationErrors(config: any): string[] {
         if (accounts !== "remote") {
           errors.push(
             getPrivateKeyError(
-              accounts,
+              0,
```

Comment:
I think I would use a custom error message here instead of using `getPrivateKeyError` with a 0 index.